src/ui/simulator/toolbox/components/htmllistbox/item/thermal-cluster-item.cpp

```diff
@@ -38,36 +38,54 @@ namespace HTMLListbox
 namespace Item
 {
 ThermalClusterItem::ThermalClusterItem(Antares::Data::ThermalCluster* a) :
- ClusterItem(a), pThermalClusterMarketBidCost(a->marketBidCost), pThermalCluster(a)
+ ClusterItem(a), pThermalCluster(a)
 {
     preloadImages();
+    preloadAdditionalIcons();
 }
 
 ThermalClusterItem::ThermalClusterItem(Antares::Data::ThermalCluster* a,
                                        const wxString& additional) :
- ClusterItem(a, additional), pThermalClusterMarketBidCost(a->marketBidCost), pThermalCluster(a)
+ ClusterItem(a, additional), pThermalCluster(a)
 {
     preloadImages();
+    preloadAdditionalIcons();
 }
 
 ThermalClusterItem::~ThermalClusterItem()
 {
 }
 
-wxString ThermalClusterItem::getClusterIconFilePath()
+void ThermalClusterItem::preloadAdditionalIcons()
 {
-    return getIconFilePath("images/16x16/thermal.png");
+    pIconFileRefresh = getIconFilePath("images/16x16/G.png");
+    pIconFileNoRefresh = getIconFilePath("images/16x16/NG.png");
 }
 
 wxString ThermalClusterItem::htmlContentTail()
 {
     wxString tail_to_return;
-    tail_to_return << Math::Round(pThermalClusterMarketBidCost, 3)
+    tail_to_return << Math::Round(pThermalCluster->marketBidCost, 3)
```

Comment:
I totally agree with that change : market bid cost is not a **ThermalClusterItem**'s data member anymore.